doc/api/http.md

```diff
@@ -377,13 +377,13 @@ const proxy = http.createServer((req, res) => {
 proxy.on('connect', (req, cltSocket, head) => {
   // Connect to an origin server
   const { port, hostname } = new URL(`http://${req.url}`);
-  const srvSocket = net.connect(port || 80, hostname, () => {
+  const serverSocket = net.connect(port || 80, hostname, () => {
     cltSocket.write('HTTP/1.1 200 Connection Established\r\n' +
                     'Proxy-agent: Node.js-Proxy\r\n' +
                     '\r\n');
-    srvSocket.write(head);
-    srvSocket.pipe(cltSocket);
-    cltSocket.pipe(srvSocket);
+    serverSocket.write(head);
+    serverSocket.pipe(cltSocket);
```

Comment:
I don't object, but maybe you should go the other way, not the pleasing symetricality of `srvSocket` and `cltSocket`? Or perhaps `clt` needs changing to `client` at the same time?